docs/getting-started/integrating-codacy-with-visual-studio-code.md

```diff
@@ -0,0 +1,90 @@
+# Integrating Codacy with Visual Studio Code
+
+If you're using Visual Studio Code, integrating Codacy into your setup can enhance your workflow, provide valuable insights into your code, and streamline your code review process without leaving the editor.
+
+To integrate Codacy into your setup, install the Codacy extension for VS Code by following the instructions below. This extension notifies you about the quality of a pull request, highlights problematic code patterns, and displays code quality metrics, all within VS Code.
```

Comment:
I'm not sure we want to abbreviate Visual Studio Code to VS Code. Is there any reference of known extensions / brands or even themselves using the abbreviation in formal documentation?

Reply:
Seems like it's generally acceptable:

* https://marketplace.visualstudio.com/items?itemName=Postman.postman-for-vscode
* https://marketplace.visualstudio.com/items?itemName=Codecov.codecov
* https://marketplace.visualstudio.com/items?itemName=snyk-security.snyk-vulnerability-scanner